brave/brave-core
# Replaced lint-suppressions.xml.patch with python chromium_src override

Resolves https://github.com/brave/brave-browser/issues/42901



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
Not supposed for QA, but for devs mostly
1. Run regular build - should succeed
2. Run the same build - should succeed and should be fast and report nothing changed
3. Edit `chromium_src/chrome/android/expectations/lint-suppressions.xml` with some white spaces
4. Run build - it should succeed but but  should take some time
5. Comment out issues at `chromium_src/chrome/android/expectations/lint-suppressions.xml`, run build - it should fail.
